client/src/sessions/password.rs

```diff
@@ -0,0 +1,34 @@
+use super::session::Session;
+use tpm2_rs_base::errors::{TssResult, TssTcsError};
+use tpm2_rs_base::{
+    Tpm2bAuth, Tpm2bNonce, Tpm2bSimple, TpmaSession, TpmiShAuthSession, TpmsAuthCommand,
+    TpmsAuthResponse,
+};
+
+/// A password session.
+#[derive(Debug, PartialEq, Default)]
+pub struct PasswordSession {
+    pub auth: Tpm2bAuth,
```

Comment:
Nit: I do wonder if we need this to be `pub`, I don't see us needing to have mutable access to the inner `auth` value. Might be nicer to just have a construction function. I think the closer we can get to something like `PasswordSession::new("my_password")`, probably the nicer it'll be to use.

Reply:
I actually strongly agree, but would it be ok if w do this separately, ? as I didn't write create `PasswordSession` here.

Reply:
Yeah that's fine, thanks!